python/lsst/ap/association/packageAlerts.py

```diff
@@ -389,11 +389,12 @@ def createCcdDataCutout(self, image, skyCenter, extent, photoCalib, srcId):
             CCDData object storing the calibrate information from the input
             difference or template image.
         """
+        point = image.getWcs().skyToPixel(skyCenter)
```

Comment:
Wouldn't it be better to pass in the pixel centroid to this method from the diaSource record? No need to do WCS math at all, since we know where the source centroid is.

Reply:
Can we trust that the (x,y) of the DIASource are the same on the calexp and matched template?

Reply:
Yes, those are made to be 1-1 pixel aligned (with the template having a buffer, but the xy0 set such that they are aligned).